pkg/harbor/model.go

```diff
@@ -34,6 +35,16 @@ var severityToString = map[Severity]string{
 	SevCritical:   "Critical",
 }
 
+var stringToSeverity = map[string]Severity{
+	"Unknown":    SevUnknown,
+	"None":       SevNone,
```

Comment:
`None` is defined in Clair side, right? In Harbor code, no `None` is defined. How should we handle this case?

Reply:
<img width="934" alt="Screen Shot 2019-11-06 at 13 03 00" src="https://user-images.githubusercontent.com/5753287/68269740-e1676c00-0095-11ea-8f48-b47ca5d1001d.png">

Which version of CVSS we will follow?

Reply:
@danielpacak 

Any opinions of this question?

Reply:
I wish I have called `Negligible` severity as `None` in the API spec when we were defining it. It would be fully compliant with CVSS 3.0 ratings.

<img width="991" alt="severity_rating" src="https://user-images.githubusercontent.com/1322923/68380181-9f026580-014f-11ea-94da-142636dd8f76.png">

For now we have two options:
* We keep `Neglibile` severity as synonym to `None` in CVSS 3.0 nomenclature and I'll remove None enum from Clair adapter - simple but not consistent with CVSS 3.0
* We rename `Neglibile` to `None` in API spec and Harbor code - painful but consistency pays off

WDYT @steven-zou

Reply:
@danielpacak 

Let me think carefully and check if we have time to do the refactor.